CHANGELOG.md

```diff
@@ -69,6 +69,9 @@
 - PR #4845 Add CUDF_FUNC_RANGE to top-level cuIO function APIs
 - PR #4848 Side step `unique_count` calculation in `scatter_by_map`
 - PR #4864 Add support for `__array__` method in cuDF
+- PR #4865 Add docs and clarify limitations of `applymap`
```

Comment:
Can you fix line 48 to put `_metadata` in single ticks instead of quotes to fix the markdown formatting?

Reply:
Done 👍